<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plot1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>571</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>371</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Obiekt</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="k">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dA">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>k</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dA</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="alfa">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Î±</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Regulator</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="kP">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>k</string>
      </property>
     </widget>
     <widget class="QPushButton" name="zastosujP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Zastosuj</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Regulator P</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxPID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>291</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Regulator PID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>k</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="kPID">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="tiPID">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="tdPID">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="nPID">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="bPID">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="tpPID">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ti</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Td</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tp</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxGPC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Regulator GPC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Generator</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Wzmocnienie:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="stala_wzmocnienie">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Skok:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="impuls_skok">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Amplituda:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="pr_amplituda">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Okres:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="pr_okres">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>WypeÅnienie:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="pr_wypelnienie">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Amplituda:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sinus_amplituda">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Okres:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sinus_okres">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Amplituda:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="trojkat_amplituda">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="trojkat_okres">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>380</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>380</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Okres:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="szum_wariancja">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>460</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Wariancja:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="stala">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>StaÅa</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="impuls">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Impuls</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="prostokat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fala prostokÄtna</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="sinus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sinus</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="trojkat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TrÃ³jkÄt</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="szum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Szum</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Zastosuj i zapisz</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SzybkoÅÄ symulacji</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[ms]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZapiszY">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz odpowiedÅº</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WczytajPlik">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj z pliku</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>571</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="blad">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetB">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wektor A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wektor B</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="czasprzelaczenia">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Czas przeÅÄczenia pomiÄdzy obiektami</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OdpowiedÅº symulowanego ukÅadu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>WartoÅÄ sterowania ukÅadem</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
